pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -646,15 +695,18 @@ func Run(ps *pubsub.PubSub) {
 
 		case <-ctx.gc.C:
 			start := time.Now()
-			gcObjects(&ctx, rwImgDirname)
-			gcObjects(&ctx, roContImgDirname)
+			gcOldObjects(&ctx, rwImgDirname)
+			gcOldObjects(&ctx, roContImgDirname)
```

Comment:
Do we need GC for the new objects, or are they deleted immediately when zedagent unpublishes the VolumeConfig?

Reply:
Volumes are deleted immediately when zedagent unpublishes the VolumeConfig.

Reply:
But doesn't the gc also deal with the initial/unknown, which we still need to handle?
That will happen if a volume is created, then the device is rebooted, and while the device is rebooted the volume is removed in the controller. In that case we will find the volume in /persist, but the device will never see a delete of a volumeconfig.

Reply:
Added